transformer.py

```diff
@@ -133,7 +236,10 @@ def transform_single_file(file_path, output_path, chunks, servicex=None):
     if reason_bad is not None:
         with open('log.txt', 'r') as f:
             errors = f.read()
-            raise RuntimeError("Failed to transform input file " + file_path + ": " + reason_bad + ' -- errors: \n' + errors)
+            logger.error("Failed to transform input file {}: ".format(file_path) +
+                         "{} -- errors: {}".format(reason_bad, errors))
+            raise RuntimeError("Failed to transform input file {}: ".format(file_path) +
+                               "{} -- errors: {}".format(reason_bad, errors))
```

Comment:
Assign message to a variable to make it easier to change. Use [f-strings](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings) (formatted string literals) over `str.format` (applies throughout).
```suggestion
            msg = f"Failed to transform input file {file_path}: {reason_bad} -- errors: {errors}"
            logger.error(msg)
            raise RuntimeError(msg)
```

Reply:
I originally had f-strings but ripped them out in order to support python 2.7.  I'll put in a code change that uses the var